example/centos/centos-9-aarch64-image-installer.yaml

```diff
@@ -0,0 +1,572 @@
+            language: en_US.UTF-8
+        - type: org.osbuild.users
+          options:
+            users:
+              install:
+                uid: 0
+                gid: 0
+                home: /root
+                shell: /usr/libexec/anaconda/run-anaconda
+                password: ''
+              root:
+                password: ''
+        - type: org.osbuild.anaconda
+          options:
+            activatable-modules:
+              - org.fedoraproject.Anaconda.Modules.Network
+              - org.fedoraproject.Anaconda.Modules.Payloads
+              - org.fedoraproject.Anaconda.Modules.Storage
+              - org.fedoraproject.Anaconda.Modules.Users
+        - type: org.osbuild.lorax-script
+          options:
+            path: 99-generic/runtime-postinstall.tmpl
+            basearch: aarch64
+            product:
+              name: ''
+              version: ''
+        - type: org.osbuild.dracut
+          options:
+            kernel:
+              - 8-2.fk1.aarch64
+            modules:
+              - bash
+              - systemd
+              - fips
+              - systemd-initrd
+              - modsign
+              - nss-softokn
+              - i18n
+              - convertfs
+              - network-manager
+              - network
+              - ifcfg
+              - url-lib
+              - drm
+              - plymouth
+              - crypt
+              - dm
+              - dmsquash-live
+              - kernel-modules
+              - kernel-modules-extra
+              - kernel-network-modules
+              - livenet
+              - lvm
+              - mdraid
+              - qemu
+              - qemu-net
+              - resume
+              - rootfs-block
+              - terminfo
+              - udev-rules
+              - dracut-systemd
+              - pollcdrom
+              - usrmount
+              - base
+              - fs-lib
+              - img-lib
+              - shutdown
+              - uefi-lib
+              - nvdimm
+              - prefixdevname
+              - prefixdevname-tools
+              - anaconda
+              - rdma
+              - rngd
+              - multipath
+              - fcoe
+              - fcoe-uefi
+              - iscsi
+              - lunmask
+              - nfs
+            add_drivers:
+              - cuse
+              - ipmi_devintf
+              - ipmi_msghandler
+            install:
+              - /.buildstamp
+        - type: org.osbuild.selinux.config
+          options:
+            state: permissive
+    - name: rootfs-image
+      build: name:build
+      stages:
+        - type: org.osbuild.mkdir
+          options:
+            paths:
+              - path: /LiveOS
+        - type: org.osbuild.truncate
+          options:
+            filename: /LiveOS/rootfs.img
+            size: '5368709120'
+        - type: org.osbuild.mkfs.ext4
+          options:
+            uuid: 2fe99653-f7ff-44fd-bea8-fa70107524fb
```

Comment:
This is also an interesting case, it seems images is hardcoding this, might be worth adding a comemnt

Reply:
To say 'needs' to be hardcoded?

Reply:
Well, hardcoding filesystem UUIDs is something unusual, I think it deserves a comment and we also do that in e.g. https://github.com/osbuild/otk/blob/main/example/centos/common/partition-table/x86_64.yaml#L24

So maybe something similar here: `# we use hardcoded uuids for compatibility with "images"` ?

Reply:
Addressed in separate commit.